app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
   <!-- TODO(#56): Reenable landscape support. -->
 
   <uses-permission android:name="android.permission.INTERNET" />
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Nit: Add one blank line below this. 
`Option+Cmd+L` to reorganise everything in this file.